src/v1/bookgroup/SimpleGroup.tsx

```diff
@@ -0,0 +1,41 @@
+import Image from 'next/image';
+
+interface SimpleGroupProps {
+  title: string;
+  isOwner: boolean;
+  imageSource: string;
+  eventHandler: () => void;
+}
+
+const SimpleGroup = ({
+  title,
+  isOwner,
+  eventHandler,
+  imageSource,
+}: SimpleGroupProps) => {
+  return (
+    <div
+      className="flex h-[15rem] w-[10rem] flex-col gap-[0.5rem]"
```

Comment:
p1;

```suggestion
      className="flex h-[14.9rem] w-[9.9rem] flex-col gap-[0.5rem]"
```

피그마에 1px 단위까지 똑같이 적용해 주셔야합니다!!